cs50/2017/fall/recover/check50/__init__.py

```diff
@@ -54,6 +55,59 @@ class Recover(Checks):
         "3510482b20a5b93eb8e8558227fa80f8460581f71e433932d11bac989e9d17fc"   # 049.jpg
     ]
```

Comment:
Per comment below, would you mind moving these hashes to a file too, as with:

```
md5sum *.jpg
```

That'll make these much easier to update.